src/cloudant/design_document.py

```diff
@@ -39,11 +39,17 @@ class DesignDocument(Document):
         either a ``CouchDatabase`` or ``CloudantDatabase`` instance.
     :param str document_id: Optional document id.  If provided and does not
         start with ``_design/``, it will be prepended with ``_design/``.
+    :param bool partitioned: Optional. Create as a partitioned design document.
+        Defaults to ``False``.
```

Comment:
Again this doesn't match the server side default (which I believe is variable depending on the type of the DB). That's OK, but I wanted to call it out.